engine/xml-config/src/main/java/org/datacleaner/configuration/JaxbConfigurationReader.java

```diff
@@ -683,6 +684,24 @@ private ReferenceDataCatalog createReferenceDataCatalog(final ReferenceDataCatal
                         synonymCatalogList.add(sc);
 
                         removeVariablePath();
+                    } else if (synonymCatalogType instanceof ValueListSynonymCatalogType) {
+                        final ValueListSynonymCatalogType vlsct = (ValueListSynonymCatalogType) synonymCatalogType;
```

Comment:
I believe I'm just following the existing code style in this file. I agree it's overly terse, though.